lib/api/accounts.js

```diff
@@ -35,48 +35,94 @@ module.exports = function (app) {
 		this.validPublicKey = publicKey =>
 			(typeof publicKey === 'string' && publicKey.match(/^([A-Fa-f0-9]{2}){32}$/g));
 
-		this.getAccount = (address, cb) =>
+		this.getAccount = (address, cb) => {
 			request.get({
-				url: `${app.get('lisk address')}/api/accounts?address=${address}`,
+				url: `${app.get('lisk address')}/accounts?address=${address}`,
 				json: true,
 			}, (err, response, body) => {
 				if (err || response.statusCode !== 200) {
 					return cb(err || 'Response was unsuccessful');
-				} else if (body.success === true) {
-					const account = body.account;
-					account.knowledge = knowledge.inAccount(account);
-					return cb(null, account);
+				} else if (body && Array.isArray(body.data)) {
+					if (body.data.length > 0) {
+						const account = body.data[0];
+						account.knowledge = knowledge.inAccount(account);
+						return cb(null, account);
+					}
+					return cb(null, []);
 				}
 				return cb(body.error);
 			});
+		};
 
 		this.getAccountByPublicKey = (publicKey, cb) =>
 			request.get({
-				url: `${app.get('lisk address')}/api/accounts?publicKey=${publicKey}`,
+				url: `${app.get('lisk address')}/accounts?publicKey=${publicKey}`,
+				json: true,
+			}, (err, response, body) => {
+				if (err || response.statusCode !== 200) {
+					return cb(err || 'Response was unsuccessful');
+				} else if (body && Array.isArray(body.data)) {
+					if (body.data.length > 0) {
+						const account = body.data[0];
+						account.knowledge = knowledge.inAccount(account);
+						return cb(null, account);
+					}
+					return cb(null, []);
+				}
+				return cb(body.error);
+			});
+
+		this.getMultisignaturesGroups = (account, cb) => {
+			request.get({
+				url: `${app.get('lisk address')}/accounts/${account.address}/multisignature_groups`,
+				json: true,
+			}, (err, response, body) => {
+				if (response.statusCode === 404 &&
+					typeof body === 'object' &&
+					body.message === 'Multisignature account not found'
+				) {
+					return cb(null, account);
+				} else if (err || response.statusCode !== 200) {
+					return cb(err || 'Response was unsuccessful');
+				} else if (body && Array.isArray(body.data) && body.data.length > 0) {
+					account.multisignatures = _.extend(account.multisignatures || []);
+					return cb(null, account);
+				}
+				return cb(body.error);
+			});
+		};
+
+		this.getMultisignaturesMemberships = (account, cb) => {
+			request.get({
+				url: `${app.get('lisk address')}/accounts/${account.address}/multisignature_memberships`,
 				json: true,
 			}, (err, response, body) => {
 				if (err || response.statusCode !== 200) {
 					return cb(err || 'Response was unsuccessful');
-				} else if (body.success === true) {
-					const account = body.account;
-					account.knowledge = knowledge.inAccount(account);
+				} else if (body && Array.isArray(body.data)) {
+					if (body.data.length > 0) {
+						account.multisignatures = _.extend(
```

Comment:
please add comment what is happening here

Reply:
Underscore _.extend() merges two objects into one. In this example, we gather all members and combine them with the present data.

Reply:
Yep, I know, but would be nice to explain (as a comment in code) why we merge `account.multisignatures` with `members`